envpool/mujoco/dmc/mujoco_dmc_align_test.py

```diff
@@ -56,6 +58,10 @@ def reset_state(
         for _ in range(200):
           env.physics.step()
         env.physics.data.time = 0
+      if task == "reacher":
+        env.physics.model.geom_pos = ts.observation.geom_pos.reshape(
+          env.physics.model.geom_pos.shape
+        )
```

Comment:
If we need to change the reacher.h rather than use the reshape?

Reply:
I use target to record target_x and target_y. No need to copy the whole data.